__tests__/e2e/libs/bootstrap.js

```diff
@@ -0,0 +1,17 @@
+const data = require("../database/seed.js");
+
+const wallets = [data.wallet.id, data.walletB.id, data.walletC.id, data.walletTrustD.id, data.walletTrustE.id];
+
+exports.mochaHooks = {
+    beforeAll: async () => {
+        console.log('Creating test data in DB...');
+        await data.clear(wallets);
+        await data.seed();
+    },
+    afterAll: async () => {
+        console.log('Clearing test data from DB!');
+        await data.clear(wallets);
+    }
+};
+
+exports.testData = data;
```

Comment:
@michaltrunek Is this hook the reason that fails to connect to the local DB?
![image](https://user-images.githubusercontent.com/5744708/119292772-9e72e680-bc83-11eb-9420-e32541648c1a.png)

Reply:
@dadiorchen I moved the config file into the e2e folder, now should work. Can you please try to pull latest and run again?

Reply:
Oh, I didn't realized that we need to approve it everytime, let me figure out how to it happen automatically.